stripe/src/main/java/com/stripe/android/exception/PollingFailedException.java

```diff
@@ -0,0 +1,29 @@
+package com.stripe.android.exception;
+
+
+/**
+ * Exception indicating that a polling request has failed to discover a change in the source.
+ */
+public class PollingFailedException extends StripeException {
+
+    private boolean mIsExpired;
+
+    /**
+     * Create a polling failure exception with a description of the failure and whether or not it
+     * expired.
+     *
+     * @param message a message explaining the cause of the failure
+     * @param isExpired whether or not the polling request expired
+     */
+    public PollingFailedException(String message, boolean isExpired) {
```

Comment:
This is a weird way to handle an error state -- that's (at the moment) by design, though I'm a little grossed out by creating an exception that I never actually throw. The reason for it is because in the synchronous version of this method, I'll throw the exception, since there won't be a callback (this is a more common way of failure for Rx network methods)